Cargo.toml

```diff
@@ -83,6 +83,8 @@ dioxus-hot-reload = { path = "packages/hot-reload", version = "0.4.0" }
 dioxus-fullstack = { path = "packages/fullstack", version = "0.4.1"  }
 dioxus_server_macro = { path = "packages/server-macro", version = "0.4.1" }
 log = "0.4.19"
```

Comment:
Is log still necessary after this shift?

Reply:
Not anymore, a few example packages were using this but it should be fixed now